packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -520,6 +527,9 @@ class RawGestureDetector extends StatefulWidget {
   /// duplication of information.
   final bool excludeFromSemantics;
 
+  /// {@macro flutter.gestures.recognizer.dragStartBehavior}
+  final DragStartBehavior dragStartBehavior;
```

Comment:
i don't think RawGestureDetector needs this, only GestureDetector does.

Reply:
I'm pretty sure it does? There are some widgets that create raw gesture detectors and for those this is needed.

Reply:
It should just be done in the initializer than RawGestureDetector gets passed. RawGestureDetector shouldn't need to know anything about any of these classes.